### DISCUSSION TITLE
🚀 Feature Request: open-vsx.org 仓库中搜不到 alova 扩展

### DISCUSSION THREAD

**Discussion by Lnncoco** [alovajs/alova#597] [Feature Request]
### What problem does the new feature need to solve?

兼容VSCode生态的编辑器使用 open-vsx.org 源, 例如 VSCodium 、 theia 或是 Codeium 的 windsurf, 可以考虑这里也上传一份?

### How important is it to you?

Better to have

**Comment by MeetinaXD** [comment]
好提议 @JOU-amjs @czhlin

**Reply by JOU-amjs** [reply]
整！

**Comment by JOU-amjs** [comment]
@Lnncoco 已处理
